packages/editor/src/components/preferences-modal/index.js

```diff
@@ -112,14 +114,16 @@ function PreferencesModalContents( { extraSections = {} } ) {
 										'Allow right-click contextual menus'
 									) }
 								/>
-								<PreferenceToggleControl
-									scope="core"
-									featureName="enableChoosePatternModal"
-									help={ __(
-										'Shows starter patterns when creating a new page.'
-									) }
-									label={ __( 'Show starter patterns' ) }
-								/>
+								{ hasStarterPatterns && (
```

Comment:
There is a comment on the reverted PR about removing the condition and always showing the preference. 
I think it makes sense, but I don't feel strongly about it. 

https://github.com/WordPress/gutenberg/pull/66836/files#r1859203782

Reply:
Thanks for the reference. I was also considering the same.